src/Chassis.Fairing.WebApi/ChassisWebApiBootstrapper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Web.Http;
+using Autofac.Integration.WebApi;
+using Chassis.Apps;
+
+namespace Chassis.Fairing.WebApi
+{
+    public static class ChassisWebApiBootstrapper
+    {
+        public static void Bootstrap(IApplication app, HttpConfiguration cfg)
+        {
+            cfg.DependencyResolver = new AutofacWebApiDependencyResolver(app.Container);

```

Comment:
This belongs in an `IWebApiStartupStep`, or move it into `WebApiStartupFeature`.